src/Middleware/HttpsPolicy/src/HttpsRedirectionMiddleware.cs

```diff
@@ -103,67 +114,55 @@ public Task Invoke(HttpContext context)
             return Task.CompletedTask;
         }
 
-        private bool TryGetHttpsPort(out int port)
+        private int TryGetHttpsPort()
```

Comment:
Comment: Returns -1 if we were unable to determine the port.

Or make a constant `PortNotFound = -1`